CheckerboardKarel.java

```diff
@@ -12,5 +12,124 @@
 public class CheckerboardKarel extends SuperKarel {
 
 	// You fill in this part
-
+	public void run(){
+		
+		int n1=8;
+		int n2=8;
+		
+		
+		for(int i=1;i<=n1;i++)
+		{
+			for(int j=1;j<=n2;j++)
+			{
+				if(i%2==0)
+				{
+					if(frontIsClear())
+					{
+						if(j%2!=0)
+						{
+							putBeeper();
+							move();
+						}
+						else
+						{
+							move();
+						}
+					}
+					else
+					{
+						turnRight();
+						move();
+						turnRight();
+					}
+				}
+				else
+				{
+					if(frontIsClear())
+					{
+						if(j%2!=0)
+						{
+							putBeeper();
+							move();
+						}
+						else
+						{
+							move();
+						}
+					}
+					else
+					{
+						turnLeft();
+						move();
+						turnLeft();
+					}
+					
+				}
+		}
+	}
+		
+		
+		/*for(int i=1; i<=n1;i++)
```

Comment:
always remove dead code... Commented code is dead code... And we should never submit any assignment with such code.